### DISCUSSION TITLE
Error flet build apk or aab -vv

### DISCUSSION THREAD

**Discussion by Razer-0** [flet-dev/flet#3398] [Q&A]
VERBOSE: Obtaining dependency information for python-dotenv>=0.13 from https://files.pythonhosted.org/packages/6a/3e/b68c118422ec867fa7ab88444e1274aa40681c606d59ac27de5a5588f082/python_dotenv-1.0.1-py3-none-any.whl.metadata VERBOSE: Using cached python_dotenv-1.0.1-py3-none-any.whl.metadata (23 kB) VERBOSE: Collecting watchfiles>=0.13 (from uvicorn[standard]<1,>=0->flet==0.21.2) VERBOSE: Using cached watchfiles-0.21.0.tar.gz (37 kB) VERBOSE: Installing build dependencies: started VERBOSE: Installing build dependencies: finished with status 'error'

**Comment by ndonkoHenri** [comment]
Can you share more info? (Your requirements.txt for ex)

**Comment by Razer-0** [comment]
Hello, thank you for responding, the code I want to compile is just a simple login window without any functionality, I still attach the code here...

```
import flet as ft


def main(page: ft.Page):
	page.title = "Flet counter example"
	page.vertical_alignment = ft.MainAxisAlignment.CENTER

	page.add(
			ft.Column(
					controls=[
						ft.Row(controls=[ft.TextField(label="Email", width=300)],
						       alignment=ft.MainAxisAlignment.CENTER),
						ft.Row(controls=[ft.TextField(label="Password", width=300, password=True)],
						       alignment=ft.MainAxisAlignment.CENTER),
						ft.Row(controls=[ft.TextButton(text="login")], alignment=ft.MainAxisAlignment.CENTER),
						ft.Row(controls=[ft.TextButton(text="Google")], alignment=ft.MainAxisAlignment.CENTER)],
					alignment=ft.MainAxisAlignment.CENTER,
					horizontal_alignment=ft.MainAxisAlignment.CENTER,
					expand=True)
	)


ft.app(main)

```

**Comment by Razer-0** [comment]
@ndonkoHenri 
 this is what is in my requirements.txt 
flet == 0.21.2

**Comment by FeodorFitsner** [comment]
Could you please use the template to submit an issue and provide a full build log.
